test/PayID/fakes/fake-pay-id-client.ts

```diff
@@ -8,10 +8,10 @@ export default class FakePayIDClient implements PayIDClientInterface {
   /**
    * @param xrpAddressResult The object that will be returned or thrown from a call to `xrpAddressForPayID`.
    */
-  constructor(private readonly xrpAddressResult: Result<string | undefined>) {}
+  constructor(private readonly xrpAddressResult: Result<string>) {}
```

Comment:
Note that this is just updating to the changes in #283 which were not reflected here since this class wasn't actually used anywhere.